src/component/project/ProjectAccess/ProjectAccessTable/ProjectAccessTable.tsx

```diff
@@ -175,8 +175,16 @@ export const ProjectAccessTable: VFC = () => {
                         .sort()
                         .reverse()[0];
                 },
-                Cell: ({ value }: { value: Date }) => (
-                    <TimeAgoCell value={value} emptyText="Never logged" />
+                Cell: ({ value, row: { original: row } }: any) => (
+                    <ConditionallyRender
+                        condition={row.type === ENTITY_TYPE.USER}
```

Comment:
If we check the accessor above, in case it's a group, there's an attempt of fetching the most recent login from a user belonging to that group. However if this is a condition we want to add, we can also simplify the accessor.

Reply:
Talked with Nico, we will leave it as is, but how could we simplify?

Reply:
If we don't care about groups last login info, I guess we can do something like:
```
accessor: (row: IProjectAccess) => {
    if (row.type === ENTITY_TYPE.USER) {
        const userRow = row.entity as IUser;
        return userRow.seenAt || '';
    }
},
```
Removing the groups logic from the accessor, since we are not displaying it anyways.

Reply:
RIght, now I understand. The logic was already working. I think lets keep it as it was then.